engines/bastion_katello/app/assets/javascripts/bastion_katello/products/discovery/views/discovery-create.html

```diff
@@ -95,48 +93,39 @@ <h4 ng-show="discovery.selected.length > 0" translate>Selected URLs:</h4>
     <div ng-repeat="repo in discovery.selected">
 
       <div class="col-sm-offset-0 bottom-padding">
-        <i class="fa fa-circle"></i>
+        <i class="fa fa-angle-right"></i>
         <span>{{  repo.url }}</span>
         <i class="fa fa-check" ng-show="repo.created"></i>
       </div>
 
-      <div class="form-group required">
-        <label class="control-label col-sm-1" translate>Name</label>
-        <div class="col-sm-4 input">
+      <div class="clearfix">
+        <div bst-form-group label="{{ 'Name' | translate }}">
           <input ng-model="repo.name"
-                 ng-disabled="creating() || repo.created"
-                 name="repo_name"
-                 type="text"
-                 class="form-control"
-                 required/>
+          ng-disabled="creating() || repo.created"
+          name="repo_name"
+          type="text"
+          class="form-control"
+          required/>
         </div>
       </div>
 
-      <div class="form-group required">
-        <label class="control-label col-sm-1" translate>Label</label>
-        <div class="col-sm-4 input">
+      <div class="clearfix">
+        <div bst-form-group label="{{ 'Label' | translate }}">
           <input ng-model="repo.label"
-                 ng-disabled="creating() || repo.created"
-                 name="repo_label"
-                 type="text"
-                 class="form-control"
-                 required/>
-        </div>
-      </div>
-    </div>
-
-    <div class="form">
-      <div class="control-group buttons">
-        <div class="input">
-          <button ng-click="createRepos()"
-                  ng-disabled="creating() || !requiredFieldsEnabled()"
-                  translate
-                  class="btn btn-primary btn-large">
-            <i class="fa fa-spinner fa-spin" ng-show="creating()"></i>
-            Create
-          </button>
+          ng-disabled="creating() || repo.created"
+          name="repo_label"
+          type="text"
+          class="form-control"
+          required/>
         </div>
       </div>
     </div>
+    <button ng-click="createRepos()"
+      ng-disabled="creating() || !requiredFieldsEnabled()"
+      translate
+      class="btn btn-primary">
+      <i class="pficon-add-circle-o inline-icon"></i>	Create
```

Comment:
Can you remove this icon?  We are moving towards not having icons in the buttons.